variables.tf

```diff
@@ -225,3 +245,9 @@ variable "parameter_group_name" {
   description = "Name of the DB parameter group to associate"
   default     = ""
 }
+
+variable "option_group_name" {
+  type        = "string"
+  description = "Name of the DB option group to associate"
+  default     = ""
```

Comment:
will it work when `default` is an empty string?
If not, we need to remove default and make it a required param.

Reply:
Yes, this will work if empty or not defined.   However, if it IS defined, it's required that the option group already exist (i.e, if the user wants to make use of one they previously created).   If it's blank, the module will create one.   Can that be documented as such, or do we need to handle that more gracefully?

Reply:
yes good point.
Please add a `__NOTE:__ ` in `README.yaml` and then rebuild README by executing these commands:

```
make init
make readme/deps
make readme
```

Reply:
Done.